src/software/thunderscope/robot_communication.py

```diff
@@ -22,8 +26,8 @@ def __init__(
         self,
         current_proto_unix_io: ProtoUnixIO,
         multicast_channel: str,
-        interface: str,
         estop_mode: EstopMode,
+        interface: str = None,
```

Comment:
Just testing locally, it doesn't seem like the `interface` argument changes the default values correctly.
Given that the `ThunderbotsConfig` proto has not been updated, once your `__first_shot` thread sends the config, it overwrites the `interface` argument value with the default `lo` that it's set to. I guess one solution could be to just remove the `--interface` argument and rely on #3225 to allow us to save settings between consecutive Thunderscope runs, and when running diagnostics, we might have to just manually change the setting every time, or we could just display the entirety of the saved `ThunderscopeConfig` with the interface.

Reply:
If the interface is provided, I ignore the first update proto that we get now